docs/guides/backends.rst

```diff
@@ -0,0 +1,184 @@
+.. _backends_guide:
+
+==============
+Backends guide
+==============
+
+Backends are the interfaces used to run :ref:`Circuits <circuits_guide>` in Orquestra Core. In this guide we'll cover differences between types of backends (``QuantumBackend`` vs ``QuantumSimulator``, ``TrackingBackend``, ``SymbolicSimulator``), how to use backends, currently integrated backends, and creating your own backend (among other topics).
+
+
+General Backend Information
+===========================
+
+The function of a backend is to send instructions to quantum hardware or to a simulator. Since there are various types of quantum hardware and simulators, different sets of instructions needs to be written and it can become complex. ``QuantumBackend`` simplifies this process by enabling the user to deploy a single set of instructions on multiple hardware and simulators simultaneously. Therefore executing code on various devices becomes easier and more manageable.
+
+Two types of backends
+---------------------
+
+``QuantumBackend`` is the base class for all backends. ``QuantumBackend``\ s are used for physical quantum hardware backends.
+
+Because ``QuantumBackend``\ s connect to a cloud provider to run remote circuits, it is necessary to specify which device you want to connect to. For example, the ``QiskitBackend`` takes in a ``device_name`` string:
+
+.. literalinclude:: /examples/backends_guide.py
+  :start-after: # QuantumBackend creation example
+  :end-before: # End QuantumBackend creation example
+
+.. _quantum_simulator:
+
+``QuantumSimulator`` is a subclass of ``QuantumBackend`` with added functionality that only simulators can provide, such as getting the wavefunction of a circuit and getting exact expectation values. Unlike ``QuantumBackend``\ s, ``QuantumSimulator``\ s usually do not require a ``device_name`` parameter. One exception to this rule is the ``QiskitSimulator``, which has :ref:`multiple options <integrated_backends>`:
+
+.. literalinclude:: /examples/backends_guide.py
+  :start-after: # QuantumSimulator creation example
+  :end-before: # End QuantumSimulator creation example
+
+For more information on how to use each type of backend please refer to the :ref:`section in this guide on various methods <backend_methods>` or to the :doc:`API documentation </api/quantum/api/backend/index>`.
+
+Testing the Backends
```

Comment:
I would move this section at the bottom, perhaps before "How to integrate your own backend".

Reply:
fixed in 4551c621e5c9c2b2295716c545eb6211b836df64 moved to be a subsection of integrating backends. I couldn't think of a (common) scenario other than integrating a new backend where this would be helpful info, feel free to correct me ;)